Cargo.lock

```diff
@@ -2474,7 +2490,7 @@ name = "wasmtime-cli"
 version = "0.12.0"
 dependencies = [
  "anyhow",
- "faerie",
+ "faerie 0.14.0",
```

Comment:
Would it be difficult to update wasmtime-cli/debug/obj to faerie 0.15.0 so that we're just using one version?

Reply:
Sorry, I failed to grep for these. Will fix them all right now!